docs/garage.md

```diff
@@ -0,0 +1,251 @@
+BENTO_DRS_S3_USE_HTTPS=true                             # HTTPS through gateway
+BENTO_DRS_S3_BUCKET="drs"                               # Created by init-garage
+BENTO_DRS_S3_REGION_NAME="garage"                       # Must match garage.toml
+BENTO_DRS_S3_ACCESS_KEY="<from-init-garage>"            # Save from init-garage output
+BENTO_DRS_S3_SECRET_KEY="<from-init-garage>"            # Save from init-garage output
+BENTO_DRS_VALIDATE_SSL=false                            # Set to false for self-signed certs
+```
+
+Restart DRS:
+
+```bash
+./bentoctl.bash restart drs
+```
+
+## Configuration Reference
+
+The `garage.toml` file is automatically generated by `init-garage` from the template at `etc/templates/garage/template.toml`. It substitutes environment variables and stores the result at `lib/garage/config/garage.toml`:
+
+All bucket access uses path-style addressing: `https://garage.bentov2.local/bucket/object`
+
+Admin API is published using the `BENTO_GARAGE_ADMIN_DOMAIN`
+
+## Verification
+
+### Verify S3 Integration
+
+Check if services are using S3:
+
+```bash
+# Check DRS logs for S3 activity
+./bentoctl.bash logs drs | grep -i s3
+
+# Check Drop-Box logs for S3 activity
+./bentoctl.bash logs drop-box | grep -i s3
+```
+
+### Using Garage Admin API
+
+You can interact with Garage's Admin API directly:
+
+```bash
+# Set admin token
+ADMIN_TOKEN="<from BENTO_GARAGE_ADMIN_TOKEN>"
+
+# Get cluster status
+curl -H "Authorization: Bearer $ADMIN_TOKEN" \
+  https://admin.garage.bentov2.local/v2/GetClusterLayout
+
+# List buckets
+curl -H "Authorization: Bearer $ADMIN_TOKEN" \
+  https://admin.garage.bentov2.local/v2/ListBuckets
+
+# List keys
+curl -H "Authorization: Bearer $ADMIN_TOKEN" \
+  https://admin.garage.bentov2.local/v2/ListKeys
+```
+
+## Managing Buckets
+
+### Create Additional Buckets
+
+You can create additional buckets using the Admin API:
+
+```bash
+ADMIN_TOKEN="<from BENTO_GARAGE_ADMIN_TOKEN>"
+ACCESS_KEY="<from init-garage>"
+
+# Create bucket
+curl -X POST \
+  -H "Authorization: Bearer $ADMIN_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{"globalAlias": "my-new-bucket"}' \
+  https://admin.garage.bentov2.local/v2/CreateBucket
+
+# Get bucket ID from response, then grant permissions
+BUCKET_ID="<bucket-id-from-response>"
+
+curl -X POST \
+  -H "Authorization: Bearer $ADMIN_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d "{
+    \"bucketId\": \"$BUCKET_ID\",
+    \"accessKeyId\": \"$ACCESS_KEY\",
+    \"permissions\": {
+      \"read\": true,
+      \"write\": true,
+      \"owner\": true
+    }
+  }" \
+  https://admin.garage.bentov2.local/v2/UpdateBucket
+```
+
+### Using S3 CLI Tools
+
+Garage is S3-compatible, so you can use standard S3 tools:
+
+```bash
+# Using AWS CLI
+aws configure set aws_access_key_id <ACCESS_KEY>
+aws configure set aws_secret_access_key <SECRET_KEY>
+aws --endpoint-url https://garage.bentov2.local s3 ls
+
+# Using s3cmd (create ~/.s3cfg first)
```

Comment:
would be nice to have instructions on how to create a non-default-named s3cfg file (like we had for MinIO) first, and then some instructions on how to specify that config.